source/vir/src/ast_util.rs

```diff
@@ -61,6 +61,16 @@ pub fn n_types_equal(typs1: &Typs, typs2: &Typs) -> bool {
 
 pub const QUANT_FORALL: Quant = Quant { quant: air::ast::Quant::Forall, boxed_params: true };
 
+// Generate a unique internal quantifier ID
+pub fn new_internal_qid(name: String) -> Option<String> {
```

Comment:
I think this should go in `vir/src/def.rs`.

Reply:
Done.